third-party-drivers/phantomjs.rst

```diff
@@ -0,0 +1,85 @@
+PhantomJSDriver
```

Comment:
You have a good point here, i'll create a phantomjsdriver.readthedocs.org page with the Driver info and link it here then.
I already have a gastonjs.readthedocs.org documentation so it makes senses to have a similar doc site for phantomjsdriver.
I'll update the PR once the new doc page is up and running.